### PR TITLE
Feat/remove default values

### PR DESCRIPTION
**Pull request by faust403**

### CODE REVIEW

File: scripts/deploy_scripts/utils.bash [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+      "idle_min_interval":60,
+      "unbond_batch_switch_time":'"$UNBOND_BATCH_SWITCH_TIME"',
+      "unbonding_safe_period":'"$UNBONDING_SAFE_PERIOD"',
+      "unbonding_period":'"$UNBONDING_PERIOD"',
+      "lsm_redeem_threshold":2,
+      "lsm_min_bond_amount":"1",
+      "lsm_redeem_max_interval":60000,
+      "bond_limit":"0",
+      "min_stake_amount":"2",
+      "icq_update_delay": 5
```
Comment (albertandrejev) [inline scripts/deploy_scripts/utils.bash:177]: Please replace these default values with variables as well

File: scripts/deploy_scripts/utils.bash [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+    "token_metadata":{
+      "description":"Drop token",
+      "display":"drop",
+      "exponent":6,
+      "name":"Drop liquid staking token",
+      "symbol":"DROP"
+    },
+    "base_denom":"'"$uatom_on_neutron_denom"'",
+    "core_params":{
+      "idle_min_interval":60,
```
Comment (albertandrejev) [inline scripts/deploy_scripts/utils.bash:168]: This one as well

File: scripts/deploy_scripts/utils.bash [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+      "puppeteer_code_id":'"$puppeteer_code_id"',
+      "staker_code_id":'"$staker_code_id"',
+      "rewards_manager_code_id":'"$rewards_manager_code_id"'
+    },
+    "remote_opts":{
+      "connection_id":"'"$neutron_side_connection_id"'",
+      "transfer_channel_id":"'"$NEUTRON_SIDE_TRANSFER_CHANNEL_ID"'",
+      "port_id":"transfer",
+      "denom":"'"$TARGET_BASE_DENOM"'",
+      "update_period":100
```
Comment (albertandrejev) [inline scripts/deploy_scripts/utils.bash:155]: here

File: scripts/deploy_scripts/utils.bash [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+    },
+    "salt":"salt",
+    "subdenom":"drop",
+    "token_metadata":{
+      "description":"Drop token",
+      "display":"drop",
+      "exponent":6,
+      "name":"Drop liquid staking token",
+      "symbol":"DROP"
```
Comment (albertandrejev) [inline scripts/deploy_scripts/utils.bash:164]: This values as well, but set default value to the current constant

File: scripts/deploy_scripts/utils.bash [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+      "lsm_redeem_threshold":2,
+      "lsm_min_bond_amount":"1",
+      "lsm_redeem_max_interval":60000,
+      "bond_limit":"0",
+      "min_stake_amount":"2",
+      "icq_update_delay": 5
+    },
+    "staker_params":{
+      "min_stake_amount":"10000",
+      "min_ibc_transfer":"10000",
```
Comment (albertandrejev) [inline scripts/deploy_scripts/utils.bash:181]: and here

File: scripts/deploy_scripts/instantiate.bash [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+NEUTRON_SIDE_PORT_ID="${NEUTRON_SIDE_PORT_ID:?Variable should explicitly specified}"
+NEUTRON_SIDE_UPDATE_PERIOD="${NEUTRON_SIDE_UPDATE_PERIOD:?Variable should explicitly specified}"
+SALT="${SALT:?Variable should explicitly specified}"
+SUBDENOM="${SUBDENOM:?Variable should explicitly specified}"
+TOKEN_METADATA_DESCRIPTION="${TOKEN_METADATA_DESCRIPTION:?Variable should explicitly specified}"
+TOKEN_METADATA_DISPLAY="${TOKEN_METADATA_DISPLAY:?Variable should explicitly specified}"
+TOKEN_METADATA_EXPONENT="${TOKEN_METADATA_EXPONENT:?Variable should explicitly specified}"
+TOKEN_METADATA_NAME="${TOKEN_METADATA_NAME:?Variable should explicitly specified}"
+TOKEN_METADATA_SYMBOL="${TOKEN_METADATA_SYMBOL:?Variable should explicitly specified}"
```
Comment (albertandrejev) [inline scripts/deploy_scripts/instantiate.bash:30]: These values can have default values, please use previous hard coded values as defaults for this variables

**Review by albertandrejev — CHANGES_REQUESTED** [review]

**Review by albertandrejev — APPROVED** [review]

**Review by ratik — APPROVED** [review]

**Pull request closed without merging**